README.md

```diff
@@ -3,30 +3,44 @@ Apache HttpComponents Client
 
 Welcome to the HttpClient component of the Apache HttpComponents project.
 
+[![Build Status](https://travis-ci.com/apache/httpcomponents-client.svg?branch=master)](https://travis-ci.com/apache/httpcomponents-client)
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.apache.httpcomponents.client5/httpclient5/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.apache.httpcomponents.client5/httpclient5)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
 Building Instructions
 ---------------------
 
-For building from source instructions please refer to BUILDING.txt.
+For building from source instructions please refer to [BUILDING.txt](./BUILDING.txt).
 
 Dependencies
 ------------
 
 HttpClient main module requires Java 7 compatible runtime and
 depends on the following external libraries:
 
-* Apache HttpComponents HttpCore
-* Apache Logging Log4j2 API
-* Apache Commons Codec
+* [Apache HttpComponents HttpCore](https://github.com/apache/httpcomponents-core)
+* [Apache Logging Log4j2 API](https://github.com/apache/logging-log4j2)
```

Comment:
I wonder where this comes from. Seems to be old. Can be dropped.

Reply:
@michael-o It is not old but wrong. The correct dependencies are
* Apache HttpComponents HttpCore
* SLF4J
* Apache Commons Codec

Reply:
These are correct now.